lucida/musicservice/MusicConfig.py

```diff
@@ -0,0 +1,94 @@
+# Music API configuration
+from pygn import Pygn
+import re
+
+
+PORT = 8089
+
+# Music API username and password
+clientID = '173703779-EA19716F8D2A73DF7ECAF522D5050BF3' # Enter your Client ID from developer.gracenote.com here
+userID = Pygn.register(clientID) # Get a User ID from pygn.register() - Only register once per end-user
+
+# Mood invert index
+mood_dic = {
+"peaceful" : "65322",
+"still" : "65322",
+"romantic" : "65323",
+"fantastic" : "65323",
+"sentimental" : "65324",
+"melting" : "65324",
+"mushy": "65324",
+"tender" : "42942",
+"soft" : "42942",
+"easygoing" : "42946",
+"happy" : "42946",
+"relaxed" : "42946",
+"leisurable" : "42946" , 
+"yearning" : "65325",
+"eager":"65325",
+"hungry": "65325",
+"desired": "65325",
+"anxious": "65325",
+"sick": "65325",
+"sophisticated" : "42954",
+"experienced": "42954",
+"sensual" : "42947",
+"lay": "42947",
+"cool" : "65326",
+"calm": "65326",
+"gritty" : "65327",
+"somber" : "42948",
+"severe": "42948",
+"blue": "42948",
+"dark": "42948",
+"lowering": "42948",
+"melancholy" : "42949",
+"sad": "42949",
+"serious" : "65328",
+"solemn": "65328",
+"brooding" : "65329",
+"reflecting": "65329",
+"fiery" : "42953",
+"passionate": "42953",
+"hotter": "42953",
+"urgent" : "42955",
+"instant": "42955",
+"defiant" : "42951",
+"challenging": "42951",
+"aggressive" : "42958",
+"invasive": "42958",
+"enterprising": "42958",
+"rowdy" : "65330",
+"clamorous" : "65330",
+"excited" : "42960",
+"active": "42960",
+"living": "42960",
+"hot": "42960",
+"energizing" : "42961",
+"motivating": "42961",
+"urging": "42961",
+"empowering" : "42945",
+"stirring" : "65331",
+"engaged": "65331",
+"lively" : "65332",
+"alive": "65332",
+"sincere": "65332",
+"upbeat" : "65333",
+"optimistic" : "65333",
+"rising": "65333",
+"other" : "42966"
+}
+
+# Scan the keyword from the dict
+def keyword_scan(question):
+	word = re.sub(r'[^a-zA-Z0-9]+', ' ', question).strip().lower()
+	word_list = word.split()
+	output = []
+	for item in word_list:
+		if item in mood_dic.keys():
+			output.append(item)
+	if len(output) == 0:
+		return ""
+	else:
+		return output[0]
+
```

Comment:
It is a good idea to keep functions/methods in a separate file from the Config files. Where do you think `keyword_scan` should be moved?